content/docs/command-reference/plots/index.md

```diff
@@ -1,20 +1,20 @@
 # plots
 
-Contains commands to visualize _plot metrics_ in structured files (JSON, CSV, or
-TSV): [show](/doc/command-reference/plots/show),
-[diff](/doc/command-reference/plots/diff).
+A set of commands to visualize and compare _plot metrics_ in structured files
```

Comment:
If we talk about `dvc plots modify` then it's not only "visualize and compare", it's also setup.

Reply:
Good catch. But I think "visualize" is general enough to catch the case of modify.

Reply:
I agree with @Suor , not sure how "visualize" can catch setup to be honest. (the issue is not big though)

Reply:
> not sure how "visualize" can catch setup

plots modify is about "setup" the display properties of the plot such as plot title, X axis label, etc. That's why I think "visualize" includes that. The concept of "set up plot metrics" to me sounds like something else. No strong opinion though.